extensions/azure-keyvault/runtime/src/main/java/io/quarkiverse/azure/keyvault/secret/runtime/config/KeyVaultSecretConfigUtil.java

```diff
@@ -0,0 +1,77 @@
+package io.quarkiverse.azure.keyvault.secret.runtime.config;
+
+import com.azure.security.keyvault.secrets.models.KeyVaultSecretIdentifier;
+
+public class KeyVaultSecretConfigUtil {
+    private static final String AZURE_KEYVAULT_PREFIX = "kv//";
+    private static final String AZURE_KEYVAULT_ENDPOINT_PREFIX = "https://";
+    private static final String AZURE_VAULT_URL_FORMAT = "https://%s.vault.azure.net/secrets/%s/%s";
```

Comment:
consider that, users may use azure-keyvault-emulator
https://hub.docker.com/r/basistheory/azure-keyvault-emulator

Reply:
This is a great catch @backwind1233. However, interoperability with `azure-keyvault-emulator` was not in our informal specification. I suggest we add a User Story to the backlog to verify this interoperability is functional.

Reply:
Write down in [quarkus-834](https://dev.azure.com/azure-redhat-projects/azure-redhat-projects/_workitems/edit/834).